tools/run_tests/xds_k8s_test_driver/kubernetes-manifests/server.deployment.yaml

```diff
@@ -21,10 +21,12 @@ spec:
       % if service_account_name:
       serviceAccountName: ${service_account_name}
       % endif
+      restartPolicy: Never
       containers:
       - name: ${deployment_name}
         image: ${image_name}
         imagePullPolicy: Always
+        restartPolicy: Never
```

Comment:
@lidizheng did you mean to remove this too?

Reply:
Good catch. Removed the pod-level `restartPolicy` field.